ipyleaflet/leaflet.py

```diff
@@ -2135,7 +2128,7 @@ def add_layer(self, layer):
         layer: Layer instance
             The new layer to add.
         """
-        if isinstance(layer, dict):
+        if isinstance(layer, (dict, xyzservices.lib.TileProvider)):
```

Comment:
Just FYI, `xyzservices.lib.TileProvider` is a subclass of `dict` so the original condition would work as well.

Reply:
done